apps/etterlatte-behandling/src/main/kotlin/behandling/BehandlingDao.kt

```diff
@@ -271,6 +271,7 @@ class BehandlingDao(private val connection: () -> Connection) {
             stmt.setString(16, opphoerAarsaker?.toJson())
             stmt.setString(17, fritekstAarsak)
             stmt.setString(18, prosesstype.toString())
+            stmt.setString(18, merknad)
```

Comment:
Skal vel være 19 her

Reply:
Oops, ja, fikset nå... Kjørte til og med tester ørten ganger i går for å være sikker på at alt fungerte, men må ha klart en liten endring i siste liten 😅

Reply:
Vi har vel/har hatt tre-fire-fem PR-ar som har toucha innom desse linjene no dei siste par dagane, så ikkje rart det blir feil 😅